decentraland/unity-renderer
# feat: smart item badges

## What does this PR change?

`Closes #1646 `

Adds a badge to smart wearables. The way to check if its a smart item is finding `game.js` content from wearable representations.

![Screenshot 2021-12-20 172533](https://user-images.githubusercontent.com/56365551/146831471-0bdce75f-5807-43d3-9db3-dbef6f0516a1.jpg)

## How to test the changes?

1. Go to: https://play.decentraland.org/?DEBUG_MODE=true&NETWORK=ropsten&WITH_COLLECTIONS=43e8f43a-f260-4360-a83a-d447116cf96d&island=I11&position=-14%2C0&realm=artemis&renderer-branch=feat/smart-item-badge
2. Press tab or click the avatar
3. Open collection section
4. Check that some of the wearables have a small _thunder_ icon on the bottom right of the container
5. Wear them like any other wearable, they should be usable as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
